docs/src/main/asciidoc/sqs.adoc

```diff
@@ -694,6 +694,20 @@ public void listen(@SnsNotificationSubject String subject, @SnsNotificationMessa
 }
 ----
 
+Since 3.4.1, you can receive the complete SNS notification with all its metadata using the `SnsNotification<T>` wrapper class. This provides access to both the message payload and all the metadata such as messageId, topicArn, subject, timestamp, and message attributes.
```

Comment:
Let's avoid using "you" - `Since 3.4.1, the complete SNS notification can be received...`